fastlane.gemspec

```diff
@@ -73,7 +73,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'pry'
   spec.add_development_dependency 'pry-byebug'
   spec.add_development_dependency 'yard', '~> 0.8.7.4'
-  spec.add_development_dependency 'webmock', '~> 1.19.0'
+  spec.add_development_dependency 'webmock', '~> 2.3.2'
```

Comment:
I see 👍  I just feel that it would be nice for reviewing because the change become more clear, but it's ok!